src/SFA.DAS.LevyTransferMatching.Infrastructure/Services/OpportunitiesService/IOpportunitiesService.cs

```diff
@@ -10,8 +10,10 @@ public interface IOpportunitiesService
         Task<List<OpportunityDto>> GetAllOpportunities();
         Task<OpportunityDto> GetOpportunity(int id);
         Task<GetContactDetailsResponse> GetContactDetails(long accountId, int pledgeId);
-        Task<ApplicationDetailsDto> GetApplicationDetails(int id);
+		Task<ApplicationDetailsDto> GetApplicationDetails(long accountId, int id);
```

Comment:
Weird tab lengths present in a couple places, doesn't show up in visual studio so maybe it's fine

Reply:
Odd, I'll try to reformat and see if it goes away